plugins/azure/fix_plugin_azure/resource/security.py

```diff
@@ -118,24 +121,69 @@ class AzureSecurityAssessment(MicrosoftResource, PhantomBaseResource):
         "assessment_status": S("properties", "status") >> Bend(AzureAssessmentStatus.mapping),
         "resource_source": S("properties", "resourceDetails", "Source"),
         "resource_id": S("properties", "resourceDetails", "ResourceId"),
+        "resource_type": S("properties", "resourceDetails", "ResourceType"),
         "additional_date": S("properties", "additionalData"),
         "azurePortalUri": S("properties", "links", "azurePortalUri"),
     }
     assessment_status: Optional[AzureAssessmentStatus] = field(default=None, metadata={'description': 'The result of the assessment'})  # fmt: skip
     resource_source: Optional[str] = field(default=None, metadata={'description': 'The source of the resource that the assessment is performed on'})  # fmt: skip
     resource_id: Optional[str] = field(default=None, metadata={'description': 'The id of the resource that the assessment is performed on'})  # fmt: skip
+    resource_type: Optional[str] = field(default=None, metadata={'description': 'The resource type'})  # fmt: skip
     additional_data: Optional[Dict[str, Any]] = field(default=None, metadata={'description': 'Additional data for the assessment'})  # fmt: skip
     subscription_issue: Optional[bool] = field(default=False, metadata={'description': 'Indicates if the assessment is a subscription issue'})  # fmt: skip
 
-    def post_process(self, builder: GraphBuilder, source: Json) -> None:
-        # mark as subscription issue, when the resource id is the same as the account id
-        if (rid := self.resource_id) and (sub := self._account):
-            self.subscription_issue = rid.split("/")[-1] == sub.id
+    def parse_finding(self, source: Json) -> Finding:
+        severity_mapping = {
```

Comment:
Consider defining it statically. You can also use a defaultdict to return medium in case of undefined.